lib/mocaccino.js

```diff
@@ -44,6 +44,11 @@ module.exports = function (b, opts) {
         return { main : pkg.browser };
       }
     });
+    var broutPath = path.relative(process.cwd(), broutFile);
```

Comment:
This change is not strictly necessary, but it's a leftover from moving things around and I felt the order is now easier to understand. It could also just be reverted again.